package.json

```diff
@@ -45,6 +45,7 @@
     "npm-package-arg": "6.0.0",
     "read": "1.0.7",
     "semver": "5.1.0",
+    "shell-quote": "^1.6.1",
```

Comment:
Non freezed dependency can lead to unexpected troubles if package will be accidentially broken in patch or minor release.
So, it will be good to install exact version.